src/components/views/emojipicker/ReactionPicker.tsx

```diff
@@ -119,9 +123,17 @@ class ReactionPicker extends React.Component<IProps, IState> {
         }
     };
 
+    private isEmojiDisabled = (unicode: string): boolean => {
+        const reaction = this.getReactions()[unicode];
+
+        if (!reaction) return false;
+        return !canRedact(this.props.mxEvent.getRoomId(), reaction);
+    };
+
     render() {
         return <EmojiPicker
             onChoose={this.onChoose}
+            isEmojiDisabled={this.isEmojiDisabled}
```

Comment:
Whether you can redact or not can change depending on your membership & power level (relative to the room) so this should probably be part of the RoomContext like similar fields already are so they can respond to changes

Reply:
The PL for redaction is no different for emoji than it is for other self-redactions